Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/JAIN.MSG

```diff
@@ -97,12 +97,11 @@
 {1004}{}{Hub}
 {1005}{}{Steel|Brotherhood of Steel|Brotherhood|BoS}
 {1006}{}{Vats}
-{1007}{}{Holy|Holy One}
+{1007}{}{Holy|Holy One|Flame|Holy Flame}
 {1008}{}{Unity}
 {1009}{}{Brotherhood}
 {1010}{}{Flame|Holy Flame}

```

Comment:
1009 and 1010 don't have 1109 and 1110 anymore.

Reply:
Should be fixed now.